testdata/supervisord.d/victoriametrics.ini

```diff
@@ -11,6 +11,9 @@ command =
 		--search.disableCache=true
 		--search.maxQueryLen=72KB
 		--search.latencyOffset=5s
+		--search.maxConcurrentRequests=32
+		--search.maxQueueDuration=90s
```

Comment:
What's the reason to fetch data after 60 seconds. 60 sec is default Grafana "Query timeout" to VM datasource.
![Screenshot_20210111_145256](https://user-images.githubusercontent.com/25184101/104185059-19e2a280-541d-11eb-90eb-91c3bd7e5701.png)

Reply:
If query timeout from Grafana to VM is 60s then I can remove line search.maxQueryDuration completly. It was set, because VM default was 30s.   


See: 
https://hub.docker.com/r/victoriametrics/victoria-metrics/
-search.maxQueryDuration duration
        The maximum duration for search query execution (default 30s)

Reply:
Let's multiply by two default values maxQueryDuration and maxQueueDuration
--search.maxQueueDuration=20s
--search.maxQueryDuration=60s

Reply:
Agree with @adivinho

Reply:
@adivinho @BupycHuk 

So summary:
--search.maxQueueDuration=20s
--search.maxQueryDuration=60s
--search.maxConcurrentRequests=16

All values doubled then defaults one. OK will change it. Thanks guys.

Reply:
let's keep --search.maxConcurrentRequests=32